src/redux/Rockets/RocketsSlice.js

```diff
@@ -0,0 +1,54 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const baseUrl = 'https://api.spacexdata.com/v4/rockets';
+export const getDataFromServer = createAsyncThunk('Rockets/getDataFromServer', async () => {
+  try {
+    const response = await fetch(baseUrl);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    return error.message;
+  }
+});
+
+const initialState = {
+  rocketData: [],
+  loading: false,
+  error: '',
+};
+const RocketsSlice = createSlice({
+  name: 'Rockets',
+  initialState,
+  reducers: {
+    reserveRocket: (state, action) => {
+      const rocket = state.rocketData.find((rocket) => rocket.id === action.payload);
+      // rocket.reserved = rocket.reserved ? false : true;
+      rocket.reserved = !rocket.reserved;
```

Comment:
- [ ] Kindly remove the commented(unused) code line 25 because it's considered good practice to have a readable code that is clean of any commented(unused) code.